Create labo.yml

pfabry opened the pull request:
Adding yml file for LABO ontology.

jamesaoverton commented:
Thanks for the PR. There are two problems that have to be fixed before we can merge:

1. The `labo.owl` PURL must point to a file in RDF/XML format (see [Principle 2](http://www.obofoundry.org/principles/fp-002-format.html)). It currently points to a GitHub HTML page for a file. That could be fixed by pointing to the "raw" file at https://raw.githubusercontent.com/OpenLHS/LABO/master/LABO.owl. However that file is in OWL Functional syntax. It's better for the published versions of the ontology to be in RDF/XML, which can be read by general RDF tools and not just OWL tools.

2. It's also important for the published file to have a version IRI that will be maintained with a PURL. See [Principle 4](http://www.obofoundry.org/principles/fp-004-versioning.html).

pfabry commented:
Hello James,

Thank you for the feedback. I just did the following:
 - added a version IRI;
 - RDF/XML output format;
 - modified the labo.yml with the new (and correct, sorry for that…) link;
 - created a labo.md file in OBOFoundry.github.io/ontology/<http://OBOFoundry.github.io/ontology/>

I hope this will meet your requirements.

Paul

Le 29 août 2019 à 10:47, James A. Overton <notifications@github.com<mailto:notifications@github.com>> a écrit :

Thanks for the PR. There are two problems that have to be fixed before we can merge:

  1.  The labo.owl PURL must point to a file in RDF/XML format (see Principle 2<http://www.obofoundry.org/principles/fp-002-format.html>). It currently points to a GitHub HTML page for a file. That could be fixed by pointing to the "raw" file at https://raw.githubusercontent.com/OpenLHS/LABO/master/LABO.owl. However that file is in OWL Functional syntax. It's better for the published versions of the ontology to be in RDF/XML, which can be read by general RDF tools and not just OWL tools.

  2.  It's also important for the published file to have a version IRI that will be maintained with a PURL. See Principle 4<http://www.obofoundry.org/principles/fp-004-versioning.html>.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/OBOFoundry/purl.obolibrary.org/pull/568?email_source=notifications&email_token=ALOX5PC646B6V6SWJ5T43ULQG7OO3A5CNFSM4IR3GGS2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOD5OXVRI#issuecomment-526219973>, or mute the thread<https://github.com/notifications/unsubscribe-auth/ALOX5PFT4ATUSF2SI3ZAIQ3QG7OO3ANCNFSM4IR3GGSQ>.

jamesaoverton commented:
I'll merge this, but I have two suggestions:

1. You should include your CC-BY 4.0 license in the published OWL file
2. I recommend using the GitHub Releases mechanism for publishing releases, rather than just a folder in the repository
